### PR TITLE
fix(scylla-bench): use newer version with fixed memory consumption

### PR DESCRIPTION
**Pull request by vponomaryov**
Currently we use v0.1.6 version of the "scylla-bench" tool.
And it uses to much memory starting with v0.1.4 version.
Bug: https://github.com/scylladb/scylla-bench/issues/84
    
It was fixed in the v0.1.7
So, switch to that new version to make "scylla-bench" use normal amount of memory avoiding OOMKills.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by vponomaryov** [comment]
The problem exists after merge of this PR: https://github.com/scylladb/scylla-cluster-tests/pull/4328

**Review by dkropachev — CHANGES_REQUESTED** [review]
Please address it on ScyllaBenchThread level

**Comment by dkropachev** [comment]
Memory consumption fixed on v0.1.7, you can switch to that version to address that problem.

**Comment by vponomaryov** [comment]
> Memory consumption fixed on v0.1.7, you can switch to that version to address this problem.

Thanks for fixing the scylla-bench. Updated the PR.

**Comment by vponomaryov** [comment]
> > Memory consumption fixed on v0.1.7, you can switch to that version to address this problem.
> 
> Thanks for fixing the scylla-bench. Updated the PR.

Tested this new version. It reduced the memory consumption, but still not enough. See comment there: https://github.com/scylladb/scylla-bench/issues/84

**Comment by vponomaryov** [comment]
Fixed the second part of the problem here: https://github.com/scylladb/scylla-bench/pull/86
So, need to merge it first, then do "v0.1.8" release and then I update this PR with the "v0.1.8" version of the scylla-bench.

**Comment by dkropachev** [comment]
> Fixed the second part of the problem here: [scylladb/scylla-bench#86](https://github.com/scylladb/scylla-bench/pull/86) So, need to merge it first, then do "v0.1.8" release and then update this PR with the "v0.1.8" version of the scylla-bench.

0.1.8 is released

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@ backup_bucket_region: ''  # use the same region as a cluster
 
 events_limit_in_email: 10
 
-scylla_bench_version: v0.1.6
+scylla_bench_version: v0.1.7
```
Comment (fgelcer) [inline defaults/test_default.yaml:175]: so it must be updated to `v0.1.8`, @dkropachev ?

Comment (dkropachev) [reply defaults/test_default.yaml:175]: Yap

**Review by dkropachev — APPROVED** [review]

**Review by enaydanov — APPROVED** [review]

**Pull request merged**